src/reducers/tests/transactionReducer.test.ts

```diff
@@ -53,18 +54,6 @@ describe('transactionReducer', () => {
       });
     });
 
-    it('should return the according transaction state for a generic address.', () => {
-      payload.unformattedReceiverAddress = '5H3ZryLmpNwrochemdVFTq9WMJW39NCo5HWFEwRtjbVtrThD';
-      const action = TransactionActionCreators.setReceiver(payload);
-      const result = transactionReducer(state, action);
-      expect(result).toEqual({
-        ...state,
-        genericReceiverAccount: '5H3ZryLmpNwrochemdVFTq9WMJW39NCo5HWFEwRtjbVtrThD',
```

Comment:
The problem here is that the receiver address _is not_ a generic address, it's a valid address...

This function will throw a `INCORRECT_FORMAT` error given invalid address, maybe we assert that the function throws instead?

Reply:
Hmm, on second though I may have misunderstood how these SS58 address' are decoded 🤔